+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Sun Dec  4 04:09:49 2022 - INFO  - Start environment preparation.
+ DNF_INSTALL postfix
+ pkgs=postfix
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs postfix --node 1 --tempfile ''
+ tmpfile2='Last metadata expiration check: 1 day, 23:53:17 ago on Fri 02 Dec 2022 04:17:03 AM EST.
Error: 
 Problem: conflicting requests
  - nothing provides libicuuc.so.67()(64bit) needed by postfix-2:3.6.4-1.an8.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages)'
+ '[' -z '' ']'
+ tmpfile='Last metadata expiration check: 1 day, 23:53:17 ago on Fri 02 Dec 2022 04:17:03 AM EST.
Error: 
 Problem: conflicting requests
  - nothing provides libicuuc.so.67()(64bit) needed by postfix-2:3.6.4-1.an8.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages)'
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Sun Dec  4 04:10:23 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Sun Dec  4 04:10:24 2022 - INFO  - Start testing...
+ systemctl start postfix.service
Failed to start postfix.service: Unit postfix.service not found.
+ CHECK_RESULT 5 0 0 'start fail'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='start fail'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'start fail'
+ LOG_ERROR 'start fail'
+ message='start fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'start fail'
Sun Dec  4 04:10:25 2022 - ERROR - start fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_postfix_configuration.sh line 31'
+ message='oe_test_postfix_configuration.sh line 31'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_postfix_configuration.sh line 31'
Sun Dec  4 04:10:25 2022 - ERROR - oe_test_postfix_configuration.sh line 31
+ return 0
+ systemctl status postfix.service
+ grep 'Active: active'
Unit postfix.service could not be found.
+ CHECK_RESULT 1 0 0 'postfix not active'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='postfix not active'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'postfix not active'
+ LOG_ERROR 'postfix not active'
+ message='postfix not active'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'postfix not active'
Sun Dec  4 04:10:26 2022 - ERROR - postfix not active
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_postfix_configuration.sh line 33'
+ message='oe_test_postfix_configuration.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_postfix_configuration.sh line 33'
Sun Dec  4 04:10:27 2022 - ERROR - oe_test_postfix_configuration.sh line 33
+ return 0
+ postconf -n
+ grep 'mail_owner = postfix'
oe_test_postfix_configuration.sh: line 34: postconf: command not found
+ CHECK_RESULT 1 0 0 'check fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check fail'
+ LOG_ERROR 'check fail'
+ message='check fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'check fail'
Sun Dec  4 04:10:28 2022 - ERROR - check fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_postfix_configuration.sh line 35'
+ message='oe_test_postfix_configuration.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_postfix_configuration.sh line 35'
Sun Dec  4 04:10:29 2022 - ERROR - oe_test_postfix_configuration.sh line 35
+ return 0
+ postconf -m
+ grep mysql
oe_test_postfix_configuration.sh: line 36: postconf: command not found
+ CHECK_RESULT 1 0 0 'check fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='check fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'check fail'
+ LOG_ERROR 'check fail'
+ message='check fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'check fail'
Sun Dec  4 04:10:30 2022 - ERROR - check fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_postfix_configuration.sh line 37'
+ message='oe_test_postfix_configuration.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_postfix_configuration.sh line 37'
Sun Dec  4 04:10:31 2022 - ERROR - oe_test_postfix_configuration.sh line 37
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Sun Dec  4 04:10:32 2022 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 4
+ test 4 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sun Dec  4 04:10:33 2022 - ERROR - failed to execute the case.
+ exit 4
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Sun Dec  4 04:10:33 2022 - INFO  - start environment cleanup.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z Last metadata expiration check: 1 day, 23:53:17 ago on Fri 02 Dec 2022 04:17:03 AM EST.
Error: 
 Problem: conflicting requests
  - nothing provides libicuuc.so.67()(64bit) needed by postfix-2:3.6.4-1.an8.riscv64
(try to add '--skip-broken' to skip uninstallable packages) ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile 'Last metadata expiration check: 1 day, 23:53:17 ago on Fri 02 Dec 2022 04:17:03 AM EST.
Error: 
 Problem: conflicting requests
  - nothing provides libicuuc.so.67()(64bit) needed by postfix-2:3.6.4-1.an8.riscv64
(try to add '\''--skip-broken'\'' to skip uninstallable packages)'
Traceback (most recent call last):
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 181, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 152, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: "Last metadata expiration check: 1 day, 23:53:17 ago on Fri 02 Dec 2022 04:17:03 AM EST.\nError: \n Problem: conflicting requests\n  - nothing provides libicuuc.so.67()(64bit) needed by postfix-2:3.6.4-1.an8.riscv64\n(try to add '--skip-broken' to skip uninstallable packages)"
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Sun Dec  4 04:10:37 2022 - INFO  - Finish environment cleanup!
